templates/Dockerfile-deb.tmpl

```diff
@@ -72,6 +77,17 @@ RUN echo 'deb http://apt.postgresql.org/pub/repos/apt/ %%release%%-pgdg main 15'
         liblz4-1 \
         libzstd1 \
         libzstd-dev \
+# below are needed for cmake and pgazure build
+        checkinstall \
+        git \
+        libtemplate-perl \
+        make \
```

Comment:
It is really strange to see an explicit installation step for a basic tool as `make` that we usually take for granted.

We already have `build-essential` listed above, and AFAIK it always depends on `make` so we can skip this one. However there is no harm in adding it to the list.